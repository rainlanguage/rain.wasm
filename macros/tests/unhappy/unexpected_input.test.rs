#[macro_use]
extern crate wasm_bindgen_utils_macros;

#[wasm_export]
struct TestStruct;

#[wasm_export]
enum TestEnum {}

#[wasm_export]
type TestType = u8;

#[wasm_export]
mod test_mod {}

#[wasm_export]
const TEST_COST: u8 = 1;

#[wasm_export]
static TEST_STATIC: u8 = 1;

fn main() {}
