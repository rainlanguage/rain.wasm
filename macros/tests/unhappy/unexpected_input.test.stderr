error: unexpected input, wasm_export macro is only applicable to impl blocks or standalone functions
 --> tests/unhappy/unexpected_input.test.rs:5:1
  |
5 | struct TestStruct;
  | ^^^^^^^^^^^^^^^^^^

error: unexpected input, wasm_export macro is only applicable to impl blocks or standalone functions
 --> tests/unhappy/unexpected_input.test.rs:8:1
  |
8 | enum TestEnum {}
  | ^^^^^^^^^^^^^^^^

error: unexpected input, wasm_export macro is only applicable to impl blocks or standalone functions
  --> tests/unhappy/unexpected_input.test.rs:11:1
   |
11 | type TestType = u8;
   | ^^^^^^^^^^^^^^^^^^^

error: unexpected input, wasm_export macro is only applicable to impl blocks or standalone functions
  --> tests/unhappy/unexpected_input.test.rs:14:1
   |
14 | mod test_mod {}
   | ^^^^^^^^^^^^^^^

error: unexpected input, wasm_export macro is only applicable to impl blocks or standalone functions
  --> tests/unhappy/unexpected_input.test.rs:17:1
   |
17 | const TEST_COST: u8 = 1;
   | ^^^^^^^^^^^^^^^^^^^^^^^^

error: unexpected input, wasm_export macro is only applicable to impl blocks or standalone functions
  --> tests/unhappy/unexpected_input.test.rs:20:1
   |
20 | static TEST_STATIC: u8 = 1;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
